packages/zoe/src/contracts/autoswap.js

```diff
@@ -35,12 +38,13 @@ const start = async (zcf, _terms) => {
   } = liquidityMint.getIssuerRecord();
   let liqTokenSupply = 0;
 
-  const { brandKeywordRecord } = zcf.getInstanceRecord();
-  Object.values(brandKeywordRecord).forEach(brand =>
-    assertNatMathHelpers(zcf, brand),
-  );
+  // We have added a new issuer to the contract, so in order to get
+  // all the brands, we must call ZCF rather than use the static
+  // brands in the terms.
+  const { brands } = zcf.getTerms();
```

Comment:
I'd extract this up at the terms as well.

Reply:
@katelynsills , Just checking: the reason @dtribble 's comment is resolved is that you need the terms after making the Liquidity issuer, not before. Right?